src/UpdatedPackagesCollector.js

```diff
@@ -151,6 +151,7 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
+    // TODO: Ugh... something about this.
```

Comment:
Oops, addressed this in a [previous reply](https://github.com/asini/asini/pull/8#r79480296).